skimage/measure/tests/test_regionprops.py

```diff
@@ -518,6 +518,13 @@ def test_regionprops_table():
                    'bbox+0': array([0]), 'bbox+1': array([0]),
                    'bbox+2': array([10]), 'bbox+3': array([18])}
 
+    out = regionprops_table(np.zeros((2, 2), dtype=int),
+                            properties=('label', 'area', 'bbox'),
+                            separator='+')
+    assert len(out) == 2
```

Comment:
I'm confused about why this is 2... Shouldn't it be 3??? (Since you requested 3 properties.)

Reply:
And actually, more, since the bbox has multiple elements... I think this should have length 6!

Reply:
And also, could you please make this into a new test, "test_table_empty_image", for example?

Reply:
The resulting dictionary should have the exact same number of columns (keys), and of the same dtype (whether that is float, int, or obj), as the dictionary that would occur if the image had at least one object.

Reply:
if bbox is an object column, then that's 3 columns. But the tests above suggest that it is not an object column?